config/routes.rb

```diff
@@ -2,6 +2,8 @@
   devise_for :players
   root :to => 'static_pages#index'
 
+  resources :games
```

Comment:
Unless you have created actions for all the RESTful routes, I think it's wise to only add them as you need them. So this should look more like `resources :games, only: [:show]`